src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,31 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 5.x` to `io.sentry:sentry-android 6.0.0`
+
+* Kotlin plugin is upgraded to `1.5`.
+* Kotlin `languageVersion` is upgraded to `1.4`.
+* `Gson` is removed as a transitive dependency and vendored in the SDK.
+  * Protocol classes now implement the `JsonSerializable` and `JsonDeserializer` interfaces.
```

Comment:
Because this is under migration, I feel the assumption is that users must follow steps. So for items like this it could be good to add a heading "Note on major changes".

If there are no steps the user must follow let's point that out:

"no major api change in this version but a few bevahior and other relevant changes worth mentioning:"

Reply:
Users don't need to do anything, we don't have any other entry with this breakdown of "API changes" or "behavior changes", it's not a bad idea tho.

If you look down this page, you see that we have code snippets "Old way / New way" when they need to follow steps.